spec/support/vcr_config.rb

```diff
@@ -47,4 +47,13 @@ def test_docker_uri
     uri2.query = nil
     uri1.to_s.gsub(/\/\d{3}/,"") == uri2.to_s.gsub(/\/\d{3}/,"")
   end
+
+  config.register_request_matcher :s3_image_matcher do |request1, request2|
+    uri1 = URI(request1.uri)
+    uri2 = URI(request2.uri)
+
+    uri_regex = /^\/artworks\/(original|thumb)\/_hey-hey-[0-9]+__[A-z0-9]+\.png/
+  
+    uri1.host == uri2.host && uri_regex.match?(uri1.path) && uri_regex.match?(uri2.path)
```

Comment:
2 tiny things:
- why do you need to check that the host of one is equal to the other? since you have `match_requests_on: [... :host ... ] ` in your spec already, I don't think you need to worry about the host here
- this condition checks that `uri1` and `uri2` match the regex.  but suppose both uri1 and uri2 return the same url?  that would actually be incorrect, but your condition would return `true`.  I think you need to check that the 2 uri2 are different and that one has `/original/...` and the other has `/thumb/...`